workalendar/precomputed_astronomy.py

```diff
@@ -0,0 +1,36 @@
+"""
+Astronomical functions
+
+Computed years spread from 30 years before and after the release year.
+"""
+from datetime import date
+from gzip import open
+from json import loads
+from pathlib import Path
+from typing import Tuple
```

Comment:
Same here... Except for typing 😅 . Importing the typing module is the exception: it usually makes the code LESS readable, so the way you did it is 🆗